docs/crashlytics/overview.mdx

```diff
@@ -54,7 +54,7 @@ dependencies {
 }
 
 // This must appear at the bottom of the file
-apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.gms.google-services' // If you use Dart-only installation don't apply this line
```

Comment:
```suggestion

```

Reply:
I think it should be removed now as the documentation context refers to dart initialisation. To use the previous initialisation method, users would use [these instructions](https://firebase.flutter.dev/docs/manual-installation/android) I believe.

Reply:
yep, i think you are right